File: DevExpress-Demo/VB/ControlsDemo.Wpf/Modules/Wizard.WpfTour/MainViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO

Namespace ControlsDemo.Wizard.WpfTour

    Public Class MainViewModel

        Private WizardViewModel As WizardViewModel

        Public Sub LaunchWizard()
            WizardViewModel = ViewModelSource.Create(Of WizardViewModel)()
            WizardViewModel.SetParentViewModel(Me)
            GetRequiredService(Of IDialogService)().ShowDialog(MessageButton.OKCancel, "Wizard Control Feature Tour", WizardViewModel)
        End Sub
    End Class
End Namespace

File: DevExpress-Demo/CS/ControlsDemo.Wpf/Modules/Wizard.WpfTour/MainViewModel.cs
using System.Collections.ObjectModel;
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;

namespace ControlsDemo.Wizard.WpfTour {
    public class MainViewModel {
        WizardViewModel WizardViewModel;
        public void LaunchWizard() {
            WizardViewModel = ViewModelSource.Create<WizardViewModel>();
            WizardViewModel.SetParentViewModel(this);
            this.GetRequiredService<IDialogService>().ShowDialog(MessageButton.OKCancel, "Wizard Control Feature Tour", WizardViewModel);
        }
    }
}
